src/main/java/com/jcabi/github/StatusState.java

```diff
@@ -0,0 +1,5 @@
+package com.jcabi.github;
+
+public enum StatusState {
```

Comment:
I think this is quite a different situation. `Issue` class does not seem to leak those fields into outer (not library) classes. But this Status object is supposed to be created by library's users - thus encapsulating those into a simple enum is much more descriptive than having to look into a specific class just to get that value.

Also, don't you think the statuses in `Issue` class should be `final static`?